templates/ai/_sidebar.html

```diff
@@ -1,8 +1,9 @@
 <div class="drawer-side">
     <label for="my-drawer-2" class="drawer-overlay"></label>
     <!-- <button id="sidebar-close-btn" class="drawer-button">X</button> -->
-    <a href="/ai/home.html"><img id='sidebar-logo' src="/img/mozilla-only.png"/></a>
+    
     <ul class="menu">
+      <li><a href="/ai/home.html"><img class="dark:[filter:invert(1)]" id='sidebar-logo' src="/img/mozilla-only.png"/></a></li>
```

Comment:
I think Lighthouse/axe flagged this on Netlify build for not having an `alt=` or `arial-label=` for screen readers.